File: 251220/100점 도달하기/reach-100-point.js
const fs = require('fs');
let score = Number(fs.readFileSync(0).toString().trim());

let result = '';
let grade = 'F';
while (score <= 100) {
    if (score >= 90) {
        grade = 'A';
    } else if (score >= 80) {
        grade = 'B';
    } else if (score >= 70) {
        grade = 'C';
    } else if (score >= 60) {
        grade = 'D';
    }
    result += grade + ' ';
    score++;
}

console.log(result);

File: 251222/100점 도달하기/reach-100-point.py
n = int(input())

grade = 'F'
while n <= 100:
    if n >= 90: grade = 'A'
    elif n >= 80: grade = 'B'
    elif n >= 70: grade = 'C'
    elif n >= 60: grade = 'D'
    print(grade, end=' ')
    n += 1